examples/instrumentation-quickstart/docker-compose.yaml

```diff
@@ -21,7 +21,6 @@ services:
       - OTEL_EXPORTER_OTLP_ENDPOINT=http://otelcol:4317
       - OTEL_SERVICE_NAME=otel-quickstart-spring-boot
       - OTEL_METRIC_EXPORT_INTERVAL=5000
-      - GOOGLE_CLOUD_PROJECT
```

Comment:
Interesting - was this not supposed to have a value ?

Reply:
not sure.  Maybe it implicitly means "pass through the GOOGLE_CLOUD_PROJECT env var"?